README.md

```diff
@@ -126,23 +125,41 @@ and with [a variety of other package managers](https://docs.astral.sh/ruff/insta
 
 ### Usage
 
-To run Ruff, try any of the following:
+To run Ruff as a linter, try any of the following:
 
 ```shell
-ruff check .                        # Lint all files in the current directory (and any subdirectories)
-ruff check path/to/code/            # Lint all files in `/path/to/code` (and any subdirectories)
-ruff check path/to/code/*.py        # Lint all `.py` files in `/path/to/code`
-ruff check path/to/code/to/file.py  # Lint `file.py`
+ruff check .                        # Lint all files in the current directory (and any subdirectories).
+ruff check path/to/code/            # Lint all files in `/path/to/code` (and any subdirectories).
+ruff check path/to/code/*.py        # Lint all `.py` files in `/path/to/code`.
+ruff check path/to/code/to/file.py  # Lint `file.py`.
+ruff check @file_paths.txt          # Lint using an input file, treating its contents as newline-delimited command-line arguments.
```

Comment:
nit: `file_paths.txt` seems limited but I think it's not?
```suggestion
ruff check @ruff_args.txt          # Lint using an input file, treating its contents as newline-delimited command-line arguments.
```

Or, `cli_args`, `arguments`, etc.